zerver/management/commands/delete_deactivated_realms.py

```diff
@@ -0,0 +1,11 @@
+from typing import Any
+
+from zerver.actions.realm_settings import delete_deactivated_realms
+from zerver.lib.management import ZulipBaseCommand
+
+
+class Command(ZulipBaseCommand):
+    help = """Clears data of deactivated realms."""
+
+    def handle(self, *args: Any, **options: Any) -> None:
+        delete_deactivated_realms()
```

Comment:
As noted elsewhere, `scrub_deactivated_realms` is a better name than `delete_deactivated_realms`